tests/kafkatest/services/connect.py

```diff
@@ -252,6 +256,14 @@ def _rest_with_retry(self, path, body=None, node=None, method="GET", retries=40,
     def _base_url(self, node):
         return 'http://' + node.account.externally_routable_ip + ':' + str(self.CONNECT_REST_PORT)
 
+    def append_to_environment_variable(self, envvar, value):
```

Comment:
what is the logic for this, including the different ways `env_opts` is being encoded?